docs/operation/troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+This section contains information to help you identify and resolve errors you may encounter.
+
+### Known errors and issues
```

Comment:
In software, "Known errors and issues" often refers to "known" issues that are bugs that the maintainer has yet to fix. "Common errors and issues" are ones they are aware users may run into but are not the fault of the library.